DesktopUI2/DesktopUI2/Views/Controls/StreamEditControls/Receive.xaml

```diff
@@ -199,7 +195,7 @@
       <StackPanel>
         <TextBlock
           IsVisible="{Binding !Commits.Count}"
-          Text="{Binding UseFe2, Converter={StaticResource Fe2TextValueConverter}, ConverterParameter='This branch has no commits. Commits are generated what data is sent to a Branch in a Stream.&#x0a;&#x0a;Please select a different Branch or Stream.'}"
+          Text="This model has no version. Versions are generated what data is sent to a model in a project.&#x0a;&#x0a;Please select a different model or project."
```

Comment:
Change in pluralisation. Was this intended?
"This branch has no commit**s**"
"This model has no version" 

(I think either is fine, just want to make sure this change was intentional